src/redux/books/books.js

```diff
@@ -0,0 +1,44 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+
+// This is a string constant that will be used to identify the action type
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = {
+  books: [],
+};
+
+// This is a function that returns an action object
+const addBook = createAction(ADD_BOOK);
```

Comment:
- [ ] Currently I am unable to add a book, kindly fix this issue

Reply:
This issue has been resolved